### DISCUSSION TITLE
Custom domain redirects to wrong server

### DISCUSSION THREAD

**Discussion by flopes89** [community/community#22883] [Other Feature Feedback, Questions, & Ideas]
<p>I might be missing something terribly obivous here, but I’m sort of stuck.</p>
<p><strong>My Goal</strong></p>
<p>My apex domain is hosted on my webhoster with A and AAAA, MX, etc. records pointing to their server.</p>
<p>The www. subdomain is hosted using GitHub Pages.</p>
<p><strong>What I did</strong></p>
<ul>
<li>
<p>Added a CNAME record for the www. subdomain to point to <a href="http://flopes89.github.io" rel="nofollow noopener">flopes89.github.io</a> (which is the repo I want to use for pages).</p>
</li>
<li>
<p>Added the custom domain (<a href="http://www.florianpeschka.de" rel="nofollow noopener">www.florianpeschka.de</a>) to that repository. This created a CNAME file in the repository, pointing to the domain</p>
</li>
</ul>
<p><strong>What happens</strong></p>
<p>But now, when I browse to <a href="http://flopes89.github.io" rel="nofollow noopener">flopes89.github.io</a>, I get redirected to <a href="http://www.florianpeschka.de" rel="nofollow noopener">www.florianpeschka.de</a> - which then shows the contents of my webshoters server. Although the CNAME clearly points to the GitHub servers.</p>
<p>I did all of this yesterday, so the DNS records are properly published, an I can see that they are when I use nslookup to check the records for the domains involved.</p>
<p>What am I missing?</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/flopes89">@flopes89</a></p>
<p>This appears to be working now. Are you still experienecing the same issues?</p>

**Comment by flopes89** [comment, marked as answer]
<p>Yes, I checked another day later and everything worked as expected. I’m not sure why, though. The DNS entries were already published when I first tried it. Yet my browser still redirected me wrongly. Another day later and he seems to have forgotten the redirect. Weird stuff.</p>
